uw_sws/enrollment.py

```diff
@@ -49,11 +50,15 @@ def get_enrollment_by_regid_and_term(regid, term):
                                 term.year,
                                 term.quarter,
                                 regid)
-    return _json_to_enrollment(get_resource(url))
+    return _json_to_enrollment(get_resource(url), url)
```

Comment:
it should raise DataFailureException with a status 404 with or without  .http-headers file
How is this 404 resources differ from the regular one? why check in _json_to_enrollment ?